poetry.lock

```diff
@@ -1407,8 +1407,13 @@ name = "parso"
 version = "0.8.1"
 description = "A Python Parser"
 category = "main"
+<<<<<<< HEAD
 optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
+=======
+optional = true
+python-versions = ">=3.6"
+>>>>>>> c8bb4db0f1562d283bbf937fbeea371816764e96
```

Comment:
you might want to double check what happened here, still figuring things out a bit with poetry @SimonBiggs